Python/UpeachyaKansakar/tasks/task2/storeToSqlite.py

```diff
@@ -0,0 +1,38 @@
+import csv
+import sqlite3
+
+def main():
+    # File path.
+    filePath = 'output/articles.csv'
+    # Database.
+    database = '/home/upeachya/training'
+
+    connect = sqlite3.connect(database)
+    cursor = connect.cursor()
+
+    # Creating a new table
+    cursor.execute("""
+        CREATE TABLE articles(
+        title text,
+        author text
+    )
+    """)
```

Comment:
You can use context manager here
with con:
con.execute(
Reference:
https://docs.python.org/2/library/sqlite3.html#using-the-connection-as-a-context-manager